x-pack/plugins/security_solution/public/management/cypress/tasks/login.ts

```diff
@@ -5,242 +5,107 @@
  * 2.0.
  */
 
-import * as yaml from 'js-yaml';
-import type { Role } from '@kbn/security-plugin/common';
 import type { LoginState } from '@kbn/security-plugin/common/login_state';
-import { getWithResponseActionsRole } from '../../../../scripts/endpoint/common/roles_users/with_response_actions_role';
-import { getNoResponseActionsRole } from '../../../../scripts/endpoint/common/roles_users/without_response_actions_role';
-import { request } from './common';
-import { getT1Analyst } from '../../../../scripts/endpoint/common/roles_users/t1_analyst';
-import { getT2Analyst } from '../../../../scripts/endpoint/common/roles_users/t2_analyst';
-import { getHunter } from '../../../../scripts/endpoint/common/roles_users/hunter';
-import { getThreatIntelligenceAnalyst } from '../../../../scripts/endpoint/common/roles_users/threat_intelligence_analyst';
-import { getSocManager } from '../../../../scripts/endpoint/common/roles_users/soc_manager';
-import { getPlatformEngineer } from '../../../../scripts/endpoint/common/roles_users/platform_engineer';
-import { getEndpointOperationsAnalyst } from '../../../../scripts/endpoint/common/roles_users/endpoint_operations_analyst';
-import {
-  getEndpointSecurityPolicyManagementReadRole,
-  getEndpointSecurityPolicyManager,
-} from '../../../../scripts/endpoint/common/roles_users/endpoint_security_policy_manager';
-import { getDetectionsEngineer } from '../../../../scripts/endpoint/common/roles_users/detections_engineer';
-
-export enum ROLE {
-  t1_analyst = 't1Analyst',
-  t2_analyst = 't2Analyst',
-  analyst_hunter = 'hunter',
-  threat_intelligence_analyst = 'threatIntelligenceAnalyst',
-  detections_engineer = 'detectionsEngineer',
-  soc_manager = 'socManager',
-  platform_engineer = 'platformEngineer',
-  endpoint_operations_analyst = 'endpointOperationsAnalyst',
-  endpoint_security_policy_manager = 'endpointSecurityPolicyManager',
-  endpoint_response_actions_access = 'endpointResponseActionsAccess',
-  endpoint_response_actions_no_access = 'endpointResponseActionsNoAccess',
-  endpoint_security_policy_management_read = 'endpointSecurityPolicyManagementRead',
+import type { Role } from '@kbn/security-plugin/common';
+import { ENDPOINT_SECURITY_ROLE_NAMES } from '../../../../scripts/endpoint/common/roles_users';
+import type { SecurityTestUser } from '../common/constants';
+import { COMMON_API_HEADERS, request } from './common';
+
+export const ROLE = Object.freeze<Record<SecurityTestUser, SecurityTestUser>>({
+  ...ENDPOINT_SECURITY_ROLE_NAMES,
+  elastic: 'elastic',
```

Comment:
can we spread KIBANA_KNOWN_DEFAULT_ACCOUNTS here? Or you want to keep it separated?

Reply:
The `const` is actually an array, thus why I did not use it here... but, thats a good point. I'm going to convert it to an object and then just spread it here so that we keep it centralized.